.github/workflows/android.yml

```
@@ -0,0 +1,72 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Decode google-services.json
+        run: echo ${{ secrets['GOOGLE_SERVICES_JSON_BASE64] }} | base64 --decode > ./app/google-services.json
```

actionlint (1.7.4)

got unexpected EOF while lexing end of string literal, expecting '''

(expression)